### PR TITLE
can go back to endpoint config from extensions and options page

### PR DESCRIPTION
**Pull request by dhchandw**
#1382

### CODE REVIEW

File: src/components/ZclEndpointCard.vue [hunk 1]
```diff
@@ -21,111 +21,85 @@ limitations under the License.
...
+                createLogoSrc(
+                  true,
+                  getDeviceCategory(deviceType[0]?.packageRef),
+                  isSelectedEndpoint
+                )
+              "
+              alt=""
+            />
+            <strong>
+              Endpoint - {{ getFormattedEndpointId(endpointReference) }}</strong
```
Comment (tbrkollar) [inline src/components/ZclEndpointCard.vue:42]: remove "Endpoint -" text and add "#" character. I think it was a rebase conflict.

File: src/components/ZclEndpointCard.vue [hunk 2]
```diff
@@ -21,111 +21,85 @@ limitations under the License.
...
-            dense
-            v-close-popup
-            size="sm"
-            icon="o_delete"
-            @click="handleDeletionDialog"
-            data-test="delete-endpoint"
-          >
-            <q-tooltip> Delete </q-tooltip>
-          </q-btn>
+      <router-link :to="'/'" class="no-decoration">
```
Comment (tbrkollar) [inline src/components/ZclEndpointCard.vue:24]: not use router-link for this. add @click="" handler and a function for this in a script tag.

File: src/components/ZclEndpointCard.vue [hunk 3]
```diff
@@ -590,5 +596,9 @@ export default {
     color: #fff;
     background: var(--q-primary);
   }
+  .no-decoration {
```
Comment (tbrkollar) [inline src/components/ZclEndpointCard.vue:599]: if you use a function it is not necessary

**Comment by dhchandw** [comment]
use this.router.push instead

File: src/components/ZclEndpointCard.vue [hunk 4]
```diff
@@ -408,6 +408,10 @@ export default {
             this.selectedReporting.push(resolvedReference)
         })
       })
+    },
+    handleEndpointCardClick(endpointReference) {
```
Comment (brdandu) [inline src/components/ZclEndpointCard.vue:412]: When do we put a function on common-mixing vs here? If more than one places calls this function?
Add comments to this method?

**Review by brdandu — CHANGES_REQUESTED** [review]

**Review by brdandu — APPROVED** [review]

**Pull request merged**